tasks/Module 5/task_2.py

```diff
@@ -0,0 +1,233 @@
+"""Hero's Adventure Game."""
+from random import randint, choice
+from dataclasses import dataclass
+from task_1 import Item, Inventory, LIGHT_WEIGHT_THRESHOLD
+
+
+FOOD = ["apple", "wine", "chicken", "bread"]
+EXCLUDE_ITEMS = ["chewed gum", "used tissue"]
+ITEMS = ["gold coin", "ruby", "dagger", "rope", "sword", "shield"] + FOOD + EXCLUDE_ITEMS
+MOVES = ["north", "south", "west", "east", "l", "r", "u", "d"]
+HERO_SYMBOL = 'H'
+ITEM_SYMBOL = 'X'
+
+
+@dataclass
+class GameItem(Item):
+    """GameItem class."""
+
+    heal: int = 0
+
+    def is_rubbish(self):
+        """Check if Item is rubbish."""
+        return self.name in EXCLUDE_ITEMS
+
+    def is_food(self):
+        """Check if Item is food."""
+        return self.name in FOOD
+
+
+class Hero:
+    """Class Hero."""
+
+    def __init__(self, row, columm):
```

Comment:
I would recommend changing arguments order to `column, row` to match arguments in `Board` and `Game` constructors.